src/components/DesignProvider/DesignProvider.js

```diff
@@ -17,16 +18,50 @@ const GlobalStyle = createGlobalStyle`
   body {
     margin: 0;
     background: ${({ theme }) => theme.colors.whiteout.lightest};
+
+    /** Set default text color and family, so inheritance works */
     font-size: 16px;
-    /* set default text color and family, so inheritance works */
     font-family: ${tokens.fonts.primary};
     color: ${({ theme }) => theme.colors.neutral.darker};
+
     text-align: left;
    }
 
   * {
     color: inherit;
   }
+
+  @font-face {
```

Comment:
I did this with Poppins originally, but Next.js gets angry if you use @import for fonts. Some vulnerability or something. Hence be building that `<Fonts />` component. A bit more annoying becaue you have to add both `DesignProvider` and `Fonts`, but the error went away.

Reply:
Oh interesting, I haven't seen it throw that yet, but will try it by updating `Fonts`